pkg/plugin/manager/manager.go

```diff
@@ -75,12 +79,20 @@ func (m *manager) RunMigrations() error {
 	if len(paths) < 1 {
 		return nil
 	}
+
 	// Init directory for temporary copies of migrations
 	err = m.setupMigrationEnv()
 	if err != nil {
 		return err
 	}
 	defer m.cleanUp()
+
+	// First run the public schema migrations
+	err = m.runPublicMigrations()
```

Comment:
One other really minor thing while we're tinkering in here - I'm a fan of _not_ reusing the name `err` but instead creating a new name for each possible error so that the compiler detects if it's unchecked. e.g. this compiles
```
err := stuff()
// non-nil error undetected
err = otherStuff()
if err != nil {
     return err
}
```
and this does not:
```
stuffErr := stuff()
otherStuffErr := otherStuff()
if otherStuffErr != nil {
     return otherStuffErr
}
```

Reply:
👍